packages/components/src/Toggle/LabelToggle/LabelToggle.component.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getTheme } from '../../theme';
+
+import css from './LabelToggle.scss';
+
+const theme = getTheme(css);
+
+/**
+ *
+ * @param {Array.<Object>} buttons - buttons that should be rendered
+ * @param {string} name - the name attribute for all inputs
+ * @param {string} checked - the checked value
+ * @param {function} onChange - callback that handle the state change
+ */
+function LabelToggle({ buttons, name, checked, onChange }) {
+	const handleChange = e => {
+		onChange(e.target.value);
+	};
+
+	return (
+		<div className={theme('radio-toolbar')}>
+			{buttons.map(button => (
+				<React.Fragment key={`radio${button.value}`}>
+					<input
+						id={`radio${button.value}`}
+						type="radio"
+						value={button.value}
+						checked={checked === button.value}
+						name={name}
+						onChange={e => handleChange(e)}
```

Comment:
Don't forget the disabled and autofocus states

Reply:
I add the disabled state (and a story attached)

I really don't know how to handle the autofocus state.
Not only as it's not present in the guidelines, but also technically cause we have the focus on the hidden radio buttons and we should highlight the containers of theses.
 Could we use `ally-focus-within` to know it ? not sure at all it's the right way to do it